src/js/shield_defs.js

```diff
@@ -105,12 +105,33 @@ export function loadShields() {
   // NORTH AMERICA
   shields["GLCT"] = {
     notext: true,
+    overrideByRef: {
+      LECT: {
+        spriteBlank: "shield_glct_lect",
+      },
+      LHCT: {
+        spriteBlank: "shield_glct_lhct",
+      },
+      LMCT: {
+        spriteBlank: "shield_glct_lmct",
+        colorLighten: Color.shields.green,
+      },
+      LSCT: {
+        spriteBlank: "shield_glct_lsct",
+      },
+    },
+  };
+  shields["GLCT:Loop"] = {
+    ...shields["GLCT"],
+    banners: ["LOOP"],
+    bannerColor: Color.shields.brown,
```

Comment:
I think this is supposed to be bannerTextColor, let's check the map and see how it's rendering.